src/EventSubscriber/RubricChosenSubscriber.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+
+namespace App\EventSubscriber;
+
+
+use App\Model\SearchData;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+
+class RubricChosenSubscriber implements EventSubscriberInterface
+{
+
+    public static function getSubscribedEvents()
+    {
+        return array(
+            FormEvents::PRE_SET_DATA => 'onPreSetData',
+        );
+    }
+
+    public function onPreSetData(FormEvent $event) {
+        /** @var SearchData $data **/
+        $data = $event->getData();
+        $form = $event->getForm();
+        $formOptions = $form->getConfig()->getOptions();
+
+        $formOptions['label'] = false;
```

Comment:
```suggestion
        $formOptions['label'] = false;
        $formOptions['mapped'] = false;
```
Probier doch mal bitte, ob dies den Getter+Setter für `selectedStatusWidget` in `SearchData` nicht überflüssig macht.